src/server.tsx

```diff
@@ -91,25 +91,40 @@ const [, fetchConfiguredArticleEpicTestsCached] = cacheAsync(
     `fetchConfiguredEpicTests_ARTICLE`,
 );
 
+const [, fetchConfiguredArticleEpicHoldbackTestsCached] = cacheAsync(
+    () => fetchConfiguredEpicTests('ARTICLE_HOLDBACK'),
+    60,
+    `fetchConfiguredEpicTests_ARTICLE_HOLDBACK`,
+);
+
 const [, fetchConfiguredLiveblogEpicTestsCached] = cacheAsync(
     () => fetchConfiguredEpicTests('LIVEBLOG'),
     60,
     `fetchConfiguredEpicTests_LIVEBLOG`,
 );
 
-const fetchCachedEpicTests: { [key in EpicType]: () => Promise<EpicTests> } = {
-    ARTICLE: fetchConfiguredArticleEpicTestsCached,
-    LIVEBLOG: fetchConfiguredLiveblogEpicTestsCached,
+const getArticleEpicTests = async (mvtId: number): Promise<Test[]> => {
+    const shouldHoldBack = mvtId % 100 === 0; // holdback 1% of the audience
+    if (shouldHoldBack) {
+        const holdback = await fetchConfiguredArticleEpicHoldbackTestsCached();
+        return holdback.tests;
+    }
+    const regular = await fetchConfiguredArticleEpicTestsCached();
+    const hardCoded = await getAllHardcodedTests();
+
+    return [...regular.tests, ...hardCoded];
```

Comment:
I think we should refactor this when both the article and liveblog epics have a holdback audience, but for now this seemed like the easiest place to put this logic